packages/database/src/models/Invoice/InvoiceItem.ts

```diff
@@ -14,7 +14,8 @@ export class InvoiceItem extends Model {
   declare productId?: string;
   declare quantity: number;
   declare note?: string;
-  declare sourceId?: string;
+  declare sourceRecordType?: string;
+  declare sourceRecordId?: string;
   declare productName: string;
   declare productPrice?: number;
   declare productCode: string;
```

Comment:
These `productName`, `productCode`, `productDiscountable` fields seem redundant? I've seem them used a bit but hoping we can just rely on the underlying `InvoiceProduct` for these. But won't remove them here, in a separate PR I think.

Reply:
Oh I guess they're for finalised invoices with products that have been deleted/marked as historic? @tcaiger do you have any thoughts on this?